src/beautifiers/ds-beautify.coffee

```diff
@@ -0,0 +1,6 @@
+"use strict"
+JSBeautify = require('./js-beautify')
+
+module.exports = class DSBeautify extends JSBeautify
```

Comment:
You could just add support to `JSBeautify` instead of making a new beautifier subclass. Thoughts?